### ISSUE TITLE
Dataset Cleanup Organization

### ISSUE THREAD

**Issue by acbart** [RealTimeWeb/datasets#91]
The following datasets need to be cleaned up:

- [ ] Elections
- [ ] Education

**Comment by RyanWhitcomb-VT** [comment]
For the weather dataset, would you like me to write a cleaning script on the data we currently have, or just clean the next chunk of weather reports?

**Comment by acbart** [comment]
Dr. Kafura weighed in on this, and I just realized you didn't get CCed in:

"I dont see this as a "big" issue but it does seem worthwhile to keep it up to date if the effort is reasonable. Alternatively, we could keep a bigger window - add the last three months to what we now have and have a six month view, eventually reaching a point where we have a year's worth of data and then keep that up to date on a rolling basis. Might be interesting to see weather across the year."

**Comment by acbart** [comment]
I agree with him - if we can ever get a year's worth of data, that would be an ideal situation. In the meantime, if it's not too much effort, update!

**Comment by acbart** [comment]
http://weather-warehouse.com/WeatherHistory/PastWeatherData_Blacksburg_Blacksburg_VA_January.html
